docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -334,61 +371,32 @@ children:
         url: Appsody-LoopBack.html
         output: 'web, pdf'
 
-  - title: 'Using Express Middleware'
-    url: Express-middleware.html
-    output: 'web, pdf'
-
-  - title: 'Serving Static Files'
-    url: Serving-static-files.html
-    output: 'web, pdf'
-
-  - title: 'Dynamically adding models, repositories, and controllers'
-    url: Dynamic-models-repositories-controllers.html
-    output: 'web, pdf'
-
-  - title: 'Uploading and Downloading Files'
-    url: File-upload-download.html
-    output: 'web, pdf'
-
-  - title: 'Validation'
-    url: Validation.html
+  - title: 'Running and Debugging Apps'
     output: 'web, pdf'
     children:
-      - title: 'Validation in REST Layer'
-        url: Validation-REST-layer.html
-        output: 'web, pdf'
-
-      - title: 'Validation in the Controller, Repository and Service Layer'
-        url: Validation-controller-repo-service-layer.html
-        output: 'web, pdf'
-
-      - title: 'Validation in ORM Layer'
-        url: Validation-ORM-layer.html
-        output: 'web, pdf'
-
-  - title: 'Self-hosted REST API Explorer'
-    url: Self-hosted-rest-api-explorer.html
-    output: 'web, pdf'
 
-  - title: 'Creating CRUD REST APIs from a model'
-    url: Creating-crud-rest-apis.html
-    output: 'web, pdf'
+    - title: 'Setting debug strings'
+      url: Setting-debug-strings.html
+      output: 'web, pdf'
 
-  - title: 'Boot and Mount a LoopBack 3 Application'
-    url: Boot-and-Mount-a-LoopBack-3-application.html
-    output: 'web, pdf'
+    - title: 'Using strong-error-handler'
+      url: Using-strong-error-handler.html
+      output: 'web, pdf'
 
-  - title: 'Running cron jobs'
-    url: Running-cron-jobs.html
+  - title: 'Other Extensions'
     output: 'web, pdf'
+    children:
+    - title: 'Running cron jobs'
+      url: Running-cron-jobs.html
+      output: 'web, pdf'
 
-  - title: 'Health check'
-    url: Health.html
-    output: 'web, pdf'
+    - title: 'Health check'
+      url: Health.html
+      output: 'web, pdf'
 
-  - title: 'Metrics for Prometheus'
-    url: Metrics.html
-    output: 'web, pdf'
+    - title: 'Metrics for Prometheus'
+      url: Metrics.html
+      output: 'web, pdf'
```

Comment:
Have we covered all extensions at https://github.com/strongloop/loopback-next/tree/master/extensions?

Reply:
Good call! 

IMO, the goal of this work is to restructure the current content. Adding missing extension docs is IMO out of scope of this PR (a scope creep), I prefer to open a follow-up issue. However, if @jannyHou prefers to add the missing pages as part of this PR, then I am fine with that.

Reply:
Oh now I realized why @raymondfeng 's list has more items :)
Thanks @bajtos @raymondfeng , I can add them in a separate PR, within the same issue. They clone the content from readme files, so I believe it's not a big effort.

Reply:
Added extensions in PR https://github.com/strongloop/loopback-next/pull/6135 :)